packages/server/src/complete/complete.ts

```diff
@@ -39,7 +39,15 @@ import { ICONS, toCompletionItemForKeyword } from './CompletionItemUtils'
 
 export type Pos = { line: number; column: number }
 
-const logger = log4js.getLogger()
+// stubbing logger to make the lib work in browser
```

Comment:
**question**: can we also remove `log4js` from `package.json` or there it is still required?

Reply:
It's used widely in the whole project and I didn't want to have to rewrite too many files